frontend/src/components/MapView/MapView.tsx

```diff
@@ -87,8 +86,8 @@ export let MapView = withStore(
       onRender={(map: MapboxGL.Map) => store.get('map') || store.set('map')(map)}
       onClick={() => removePopup(store)}
     >
-      {representativePoints.length && <GeoJSONLayer
-        data={representativePointsToGeoJSON(adequacies)(representativePoints)}
```

Comment:
Why not just memoize `representativePointsToGeoJSON`?

Reply:
I'm not sure that is better. It's better in the case of a user switching back and forth between adequacy types, but then that's at the expense of having a gigantic memory overhead if the user switches between datasets.

Reply:
You're completely right!